### PR TITLE
Prepare for release 1.1.0

### PR DESCRIPTION
**Pull request by cquiroz**
Updates for the build to do a 1.1.0 release
Note that I also updated the scala.js version and changed a bit the travis build

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,18 +11,16 @@ All types are immutable and thread-safe.
...
 [GitHub](https://github.com/typelevel/squants)
 |
-[User Forum](https://groups.google.com/forum/#!forum/squants)
-|
 [Wiki](https://github.com/typelevel/squants/wiki)
 
 [![Join the chat at https://gitter.im/garyKeorkunian/squants](https://badges.gitter.im/garyKeorkunian/squants.svg)](https://gitter.im/garyKeorkunian/squants?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 ### Current Versions
 Current Release: **1.0.0**
```
Comment (derekmorr) [inline README.md]: Shouldn't this updated to 1.1.0 ?

Comment (cquiroz) [inline README.md]: Funny ,travis worked for me. I'll fix the release version

File: README.md [hunk 2]
```diff
@@ -11,18 +11,16 @@ All types are immutable and thread-safe.
 |
 [GitHub](https://github.com/typelevel/squants)
 |
-[User Forum](https://groups.google.com/forum/#!forum/squants)
```
Comment (cquiroz) [inline README.md]: Forgot to mention this. The user forum is almost unused, I'd rather have people use gitter, Any opinions about this?

Comment (derekmorr) [inline README.md]: The gitter channel doesn't seem to get much use either.

File: README.md [hunk 3]
```diff
@@ -11,18 +11,16 @@ All types are immutable and thread-safe.
...
 
 [![Join the chat at https://gitter.im/garyKeorkunian/squants](https://badges.gitter.im/garyKeorkunian/squants.svg)](https://gitter.im/garyKeorkunian/squants?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 ### Current Versions
-Current Release: **1.0.0**
-([API Docs](https://oss.sonatype.org/service/local/repositories/releases/archive/org/typelevel/squants_2.11/1.0.0/squants_2.11-1.0.0-javadoc.jar/!/index.html#squants.package))
+Current Release: **1.1.0**
+([API Docs](https://oss.sonatype.org/service/local/repositories/releases/archive/org/typelevel/squants_2.11/1.1.0/squants_2.11-1.1.0-javadoc.jar/!/index.html#squants.package))
 
 Development Build: **1.1.0-SNAPSHOT**
```
Comment (derekmorr) [inline README.md]: This should be 1.2.0-SNAPSHOT.

Comment (cquiroz) [inline README.md]: I'd rather make another PR with updates to the snapshot version after the release

Comment (derekmorr) [inline README.md]: The link to the API docs changed to 1.2.0-SNAPSHOT, so we should revert that then.

Comment (cquiroz) [inline README.md]: Right, a fix was sent, thanks
